content/docs/tasks/security/authn-policy/index.md

```diff
@@ -1,90 +1,85 @@
 ---
-title: Basic Authentication Policy
+title: Authentication Policy
 description: Shows you how to use Istio authentication policy to setup mutual TLS and basic end-user authentication.
 weight: 10
 keywords: [security,authentication]
 aliases:
     - /docs/tasks/security/istio-auth.html
 ---
 
-Through this task, you will learn how to:
-
-* Use authentication policy to setup mutual TLS.
-
-* Use authentication policy to do end-user authentication.
+This guide covers the main tasks you might need to perform when enabling, configuring, and using Istio authentication policies. You can find out more about how Istio authentication works in the [Authentication overview](/docs/concepts/security/#authentication).
```

Comment:
This guide covers the main tasks -> This task covers the point activities
Replace rthe last sentence with:

Find out more about the underlying concepts in the [authentication overview](/docs/concepts/security/#authentication)